src/services/auth-service.ts

```diff
@@ -0,0 +1,32 @@
+import envVariables from '../config/env-variables';
+import { User } from '../models/user-model';
+import { fetchReq } from '../utils/fetch-requests';
+
+const authUrl = envVariables.authUrl();
+
+export const authService = {
+  login: async (email: string, password: string) => {
+    const resp = await fetchReq(`${authUrl}/login`,'POST', '', { email, password});
+    if (resp.status !== 200) {
```

Comment:
Since you have the same error and success response handling in all of the functions, I think it would be better to create a function (ex. handleResponse(resp)), which checks the response and returns the proper result. Of course if there are requests where the response has to be handled in an other way that's an other case, but if you have the same code multiple times, it's better to create a handler function and call it when it's needed.